libs/self-care-tasks/application/src/lib/self-care-tasks.service.ts

```
@@ -1,23 +1,21 @@
 import { Injectable, InternalServerErrorException } from '@nestjs/common';
+import { SelfCareTask, SelfCareTopic, SelfCareUserTask } from '@self-care-tasks/domain';
 
 @Injectable()
 export class SelfCareTasksService {
   constructor() {
     //TODO:  add ai service here
   }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)